### PR TITLE
renaming the daily_prayer class to prayer_info

### PR DESCRIPTION
**Pull request by amerrez**
@hamir 
this PR should replace the [previous one](https://github.com/hamir/islam-buddy/pull/2) that encountered many conflicts and was hard to fix up.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import json
 
 from flask import Flask, request, make_response
-from daily_prayer import PrayerInfo
+from prayer_info import PrayerInfo
```
Comment (hamir) [inline app.py]: Our prayer_info object below is named the same thing as our package - can we rename the variable below?

**Comment by hamir** [comment]
Btw you know you can have a local branch without pushing it to remote right?

**Comment by amerrez** [comment]
yes i know, but why would that be useful here?

File: app.py (hunk 1 above)
Comment (amerrez) [inline app.py]: done with the below commit.

**Pull request merged**